### PR TITLE
chore: stop poker on gnosis chain

### PR DESCRIPTION
**Pull request by gosuto-inzasheru**
upkeep for our only injector there was configd at https://automation.chain.link/xdai/51275457462711301170724725347664046536233518759579141567815787018122162360515

### CODE REVIEW

**Comment by gosuto-inzasheru** [comment]
test run here: https://github.com/BalancerMaxis/multisig-ops/actions/runs/10794561783/job/29939128589

**Comment by gosuto-inzasheru** [comment]
note this will allow us to turn of the zkevm poker in oz defender

File: action-scripts/brownie/scripts/pokeInjectors.py [hunk 1]
```diff
@@ -4,8 +4,7 @@
 
 
 CHAINS_TO_KEEP = [
-    #    "zkevm-main", # currently running in defender due to rpc issues
-    "gnosis-main",
+    "zkevm-main",
```
Comment (Tritium-VLK) [inline action-scripts/brownie/scripts/pokeInjectors.py:7]: The poker is permissioned, so this call will fail now.  

If we want to swap keepers, we need to update the keeper on the underlying contracts and then disable oz + enable zkEVM.

As I recall we're not over our usage limit right now and/or if we are it's not by much. 

Suggest handling zkEVM as a separate ticket/work item and making sure we dot all the Is and cross all the Ts.

**Review by Tritium-VLK — CHANGES_REQUESTED** [review]

File: action-scripts/brownie/scripts/pokeInjectors.py (hunk 1 above)
Comment (gosuto-inzasheru) [reply action-scripts/brownie/scripts/pokeInjectors.py]: moved to #1333

**Review by Tritium-VLK — APPROVED** [review]

**Comment by gosuto-inzasheru** [comment]
https://app.onchainden.com/safes/gno:0x14969B55a675d13a1700F71A37511bc22D90155a/transactions/0xb1b7f633a4a23282e8e132327c918d90d1bcb88c88234d62caa9c689e9b983c5

**Pull request merged**